com.unity.render-pipelines.universal/Runtime/2D/Renderer2D.cs

```diff
@@ -29,14 +36,23 @@ public Renderer2D(Renderer2DData data) : base(data)
             m_Render2DLightingPass = new Render2DLightingPass(data);
             m_PostProcessPass = new PostProcessPass(RenderPassEvent.BeforeRenderingPostProcessing, data.postProcessData, m_BlitMaterial);
             m_FinalPostProcessPass = new PostProcessPass(RenderPassEvent.AfterRenderingPostProcessing, data.postProcessData, m_BlitMaterial);
-            m_FinalBlitPass = new FinalBlitPass(RenderPassEvent.AfterRendering, m_BlitMaterial);
+            m_FinalBlitPass = new FinalBlitPass(RenderPassEvent.AfterRendering + 1, m_BlitMaterial);
 
             m_UseDepthStencilBuffer = data.useDepthStencilBuffer;
 
-            m_AfterPostProcessColorHandle.Init("_AfterPostProcessTexture");
-            m_ColorGradingLutHandle.Init("_InternalGradingLut");
+            // We probably should declare these names in the base class,
+            // as they must be the same across all ScriptableRenderer types for camera stacking to work.
+            k_ColorTextureHandle.Init("_CameraColorTexture");
+            k_DepthTextureHandle.Init("_CameraDepthAttachment");
+            k_AfterPostProcessColorHandle.Init("_AfterPostProcessTexture");
+            k_ColorGradingLutHandle.Init("_InternalGradingLut");
```

Comment:
As said in the comments here, @phi-lira do you think we should put these strings into the base class?

Reply:
Yes, this + we were thinking about making pipeline textures accessible to user. Something like `UniversalRenderTextureType.CameraColorTexture`.

Then you could do something like `renderer.GetTexture(UniversalRenderTextureType.CameraColorTexture); `

Reply:
What do you think about this approach?

Reply:
Sounds good.